cmd/kwild/server/build.go

```diff
@@ -210,9 +210,6 @@ func newDBOpener(host, port, user, pass string) dbOpener {
 				},
 				MaxConns: maxConns,
 			},
-			SchemaFilter: func(s string) bool {
```

Comment:
When unset it default so what we were setting anyway.  I've deprecated this field since there are numerous assumptions about "ds_", and if we set this field to any other function nothing would work.

Reply:
I dont think we should change it now, but is the "ds_" necessary? The original reason we prepended DBIDs with the "x" is so that they could be used as Postgres schemas (since Postgres schemas must start with a letter).

The only reason I bring it up is b/c some people (like idOS) have begun modifying their schemas with Postgres directly while doing migrations. If this becomes more common user behavior, it might be easier to just make schema names 1-1 with the dbid.

Not anything we need to address right now, just food for though.